susemanager-utils/susemanager-sls/salt/channels/init.sls

```diff
@@ -5,9 +5,11 @@
 {%- set is_dnf = salt['pkg.version']("dnf") %}
 
 {%- if is_dnf %}
+{%- set dnf_plugins = salt['cmd.run']("find /usr/lib -type d -name dnf-plugins -printf "%T@ %p\n" | sort -nr | cut -d " " -s -f 2- | head -n 1", python_shell=True) %}
```

Comment:
I have the feeling that the command quoting might be wrong. Please check if maybe some single quotes or escaping might be needed

Reply:
Thanks @mcalmer for pointing. I have no idea why, but it's working. For sure it's better to fix.

https://github.com/uyuni-project/uyuni/pull/4626/commits/a7ce800b566a2e04959a5936de35fd4c56c7f9db is fixing it